TicTacToeTest.py

```diff
@@ -0,0 +1,216 @@
+import sys, pygame
```

Comment:
Don't forget to add a header/docstring that gives a brief summary of what the file contains/does.